packages/clay-charts/src/ChartBase.js

```diff
@@ -24,6 +24,30 @@ const DEFAULT_COLORS = [
 	'#5FC8FF',
 ];
 
+const DEFAULT_LINE_CASSES = [
+	'bb-line-dashed-2-2',
+	'bb-line-dashed-2-3',
+	'bb-line-dashed-2-4',
+	'bb-line-dashed-3-2',
+	'bb-line-dashed-3-3',
+	'bb-line-dashed-3-4',
+	'bb-line-dashed-4-2',
+	'bb-line-dashed-4-3',
+	'bb-line-dashed-4-4',
+];
+
+const DEFAULT_PATTERNS = [
```

Comment:
This API is actually `tiles`. Why don't we rename all of this so the relation is more obvious?

Reply:
@jbalsas yes, it makes sense. I renamed patterns to tiles.